packages/nc-gui/composables/useMultiSelect/index.ts

```
@@ -714,40 +731,104 @@
             activeCell.row++
             selectedRange.startRange({ row: activeCell.row, col: activeCell.col })
             scrollToCell?.()
             editEnabled.value = false
           }
         }
+        onActiveCellChanged?.()
+        break
+      case 'Enter':
+        selectedRange.clear()
+
+        let row
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)